Fw/Time/Time.cpp

```diff
@@ -2,6 +2,10 @@
 #include <Fw/Time/Time.hpp>
 
 namespace Fw {
+#ifdef FW_TIME_SERIALIZED_SIZE
+static_assert(FW_TIME_SERIALIZED_SIZE == Fw::Time::SERIALIZED_SIZE, "FW_TIME_SERIALIZED_SIZE macro is wrong");
```

Comment:
Might also want to add a static assert for 

```
Fw::Time::SERIALIZED_SIZE == Fw::TimeValue::SERIALIZED_SIZE
```

which is the FPP type ([Fw.TimeValue](https://github.com/nasa/fprime/blob/b377be2a9fda0cfcb3896c6ed3cdfe22c0af27ff/Fw/Time/Time.fpp#L4-L9))